### PR TITLE
Update packages - React Native 0.64.0

### PR DESCRIPTION
**Pull request by timarney**
React Native + Gradle updates as well as other dev dependancies

### CODE REVIEW

File: src/navigation/DevPersistedNavigationContainer.tsx [hunk 1]
```diff
@@ -63,8 +63,8 @@ function DevPersistedNavigationContainerImpl(
 
   return (
     <NavigationContainer
-      {...others}
       key={persistKey}
+      {...others}
```
Comment (jeberhardt) [inline src/navigation/DevPersistedNavigationContainer.tsx:67]: What is the purpose of this change?

Comment (timarney) [reply src/navigation/DevPersistedNavigationContainer.tsx:67]: Updated linter says key prop needs to come before spread

**Review by jeberhardt — APPROVED** [review]
